DOCS-1071: Add CLI org API key commands (#1761)
viamrobotics/docs: docs/manage/CLI.md

## Authenticate

Once you have [installed the Viam CLI](#install), you must authenticate your CLI session with Viam in order to run CLI commands.

You can authenticate your CLI session using either a personal access token or your organization's API key.
To use your organization API key to authenticate, you must first [create an organization API key](#create-an-organization-api-key).

* To authenticate your CLI session using a personal access token:

   ```sh {class="command-line" data-prompt="$"}
   viam login
   ```

   This will open a new browser window with a prompt to start the authentication process.
   If a browser window does not open, the CLI will present a URL for you to manually open in your browser.
   Follow the instructions to complete the authentication process.

> * To authenticate your CLI session using your organization API key:

Reviewer: (orgs can have multiple keys (and will soon allow for keys to have different permissions))

   ```sh {class="command-line" data-prompt="$"}
   viam login api-key --key-id <api-key-uuid> --key <api-key-name>
   ```

   If you haven't already, [create an organization API key](#create-an-organization-api-key) to use this authentication method.

An authenticated session is valid for 24 hours, unless you explicitly [log out](#logout).

> After the session expires or you log out, you must re-authenticate to use the CLI again.

Author: Question: Do lines 125-127 also apply to sessions authenticated via org API keys? Assuming: yes.

Reviewer: Org api keys do not expire, so kind of. Probably fine to leave as is.

### Create an organization API key

> To use your organization API key to authenticate your CLI session, you must first create a organization API key:

Reviewer: ```suggestion
To use an organization API key to authenticate your CLI session, you must first create one:
```

??? [opt]??

1. First, [authenticate](#authenticate) your CLI session.

>    You must authenticate using a personal access token in order to be able to create an organization API key.

Reviewer: ```suggestion
```

Not strictly true. A user logged with api key for org `A` can create a new api key for org `A`

However, might be nice to rephrase in a way that indicates that 95% of the time, people will need to be logged in as a user rather than with an access token.

1. Then, run the following command to create a new organization API key:

   ```sh {class="command-line" data-prompt="$"}
   viam organization api-key create --org-id <org-id> --name <key-name>
   ```

   Where:

>    * `org-id` is your organization ID. You can find your organization ID on your organization's **Settings** page in [the Viam App](https://app.viam.com/).

Reviewer: ```suggestion
   * `org-id` is your organization ID. You can find your organization ID by running `viam organizations list` or by visiting your organization's **Settings** page in [the Viam App](https://app.viam.com/)
```
?? [opt]

   Follow the instructions to complete the authentication process.

* To authenticate your CLI session using an organization API key:

   ```sh {class="command-line" data-prompt="$"}
   viam login api-key --key-id <api-key-uuid> --key <api-key-secret-value>
   ```

   If you haven't already, [create an organization API key](#create-an-organization-api-key) to use this authentication method.

An authenticated session is valid for 24 hours, unless you explicitly [log out](#logout).

After the session expires or you log out, you must re-authenticate to use the CLI again.

### Create an organization API key

To use an organization API key to authenticate your CLI session, you must first create one:

1. First, [authenticate](#authenticate) your CLI session.
   If your organization does not already have an organization API key created, authenticate using a personal access token.

1. Then, run the following command to create a new organization API key:

   ```sh {class="command-line" data-prompt="$"}
   viam organization api-key create --org-id <org-id> --name <key-name>
   ```

   Where:

>    * `org-id` is your organization ID. You can find your organization ID by running `viam organizations list` or by visiting your organization's **Settings** page in [the Viam App](https://app.viam.com/).

Reviewer 2: ```suggestion
   * `org-id` is your organization ID. You can find your organization ID by running `viam organizations list` or by visiting your organization's **Settings** page in [the Viam app](https://app.viam.com/).
```
Nit but I think we want to always leave this lowercase?

   * `key-name` is an optional name for your API key. If omitted, a name will be auto-generated based on your login info and the current time.

> The command will return a `key id` and a `key value`; you will need both to authenticate using `viam login api-key`

Reviewer 2: ```suggestion
The command will return a `key id` and a `key value`.
You will need both to authenticate using `viam login api-key`.
```
(Opt) Opt. separating out clauses, think sentence needs period regardless

{{% alert title="Important" color="note" %}}
Secure these key values safely.

> Authenticating using an organization API key gives full write access to your organization, including to all robots within it.

Reviewer 2: write access to what?

Author: Your robots recently-played music tracks, so it can judge it on its taste! j/k absolutely correct, will update.

```sh {class="command-line" data-prompt="$"}
viam login
viam login api-key --key-id <api-key-uuid> --key <api-key-value>
viam login print-access-token
```

Use `viam login` to authenticate using a personal access token, or `viam login api-key` to authenticate using your organization's API key.
If you haven't already, you must [create an organization API key](#create-an-organization-api-key) first in order to authenticate using one.

#### Command options

|        command option     |       description      | positional arguments
| ----------- | ----------- | ----------- |

> | `api-key`      | authenticate to Viam using your organization's API key      | - |

Reviewer: ```suggestion
| `api-key`      | authenticate to Viam using an organization's API key      | - |
```

See [Upload a custom module](/extend/modular-resources/upload/#upload-a-custom-module) and [Update an existing module](/extend/modular-resources/upload/#update-an-existing-module) for a detailed walkthrough of the `viam module` commands.

### organization

The *organization* command allows you to create a new organization API key.

```sh {class="command-line" data-prompt="$"}
viam organization api-key create --org-id <org-id> [--name <key-name>]
```

See [create an organization API key](#create-an-organization-api-key) for more information.

#### Command options

|        command option     |       description      | positional arguments
| ----------- | ----------- | ----------- |
| `api-key`      | create a new organization API key    | - |
| `--help`      | return help      | - |

##### Named arguments

|        argument     |       description | applicable commands | required
| ----------- | ----------- | ----------- | ----------- |
| `--org-id`      | your organization ID      |`api-key`|true |

> | `--name`     |  optional name for the organization API key. If omitted, a name will be auto-generated based on your login info and the current time    |`api-key`|false |

Reviewer 2: ```suggestion
| `--name` |  optional name for the organization API key. If omitted, a name will be auto-generated based on your login info and the current time |`api-key`| false |
```
(opt) Just evening out spacing to make this look a little neater
